tap_bing_ads/__init__.py

```diff
@@ -1055,21 +1054,17 @@ async def main_impl():
     args = utils.parse_args(REQUIRED_CONFIG_KEYS)
 
     CONFIG.update(args.config)
-    customer_id, fetched_account_ids = fetch_ad_accounts(
-        access_token=refresh_access_token(), developer_token=CONFIG["developer_token"],
-    )
-    account_ids = get_account_ids(fetched_account_ids)
+
+    access_token = refresh_access_token()
+    developer_token = CONFIG["developer_token"]
+
+    customer_id = request_customer_id(access_token, developer_token)
     CONFIG.update({"customer_id": customer_id})
+    account_ids = CONFIG["account_ids"]
     STATE.update(args.state)
 
-    if args.discover:
-        do_discover(account_ids)
-        LOGGER.info("Discovery complete")
-    elif args.catalog:
-        await do_sync_all_accounts(account_ids, args.catalog)
-        LOGGER.info("Sync Completed")
-    else:
-        LOGGER.info("No catalog was provided")
+    await do_sync_all_accounts(account_ids, args.catalog)
```

Comment:
I see you deleted the discover option. If we want to update the catalog in integrations runner. Can we still do it with `--discover`?

Reply:
excellent point, keeping the discovery logic <3